ground/gcs/src/plugins/welcome/qml/main.qml

```diff
@@ -90,8 +90,8 @@ Rectangle {
 
                 WelcomePageButton {
                     baseIconName: "hitl"
```

Comment:
Yeah probably, I did consider that before I got lazy. I'll give it a look in the morning, thx.